### ISSUE TITLE
Roles claim not undefined-safe

### ISSUE THREAD

**Issue by Marty** [FusionAuth/nodebb-plugin-fusionauth-oidc#10]
I receive the following error trying to get openid to work:

```
2020-11-23T17:22:52.812Z [4567/93] - error: /auth/fusionauth-oidc/callback
TypeError: Cannot read property 'some' of undefined
    at PassportOIDC._verify (/opt/nodebb/node_modules/nodebb-plugin-fusionauth-oidc/library.js:89:118)
```

https://github.com/FusionAuth/nodebb-plugin-fusionauth-oidc/blob/a3f978040ff924f3401797f7abb258308cd85dbc/library.js#L89

**Comment by mooreds** [comment]
@Marty are you running against FusionAuth or a different OIDC server?

Thanks for calling out that line.

**Comment by mooreds** [comment]
Fixed in 1.0.4
